e2e/helpers/generated_order_helper.js

```diff
@@ -0,0 +1,70 @@
+const createBlankOrder = async (I, createOrderEventPage, order) => {
+  await createOrderEventPage.selectType(order.type);
+  await I.retryUntilExists(() => I.click('Continue'), createOrderEventPage.fields.title);
+  await createOrderEventPage.enterC21OrderDetails();
+  await I.retryUntilExists(() => I.click('Continue'), '#judgeAndLegalAdvisor_judgeTitle');
+  await createOrderEventPage.enterJudgeAndLegalAdvisor(order.judgeAndLegalAdvisor.judgeLastName, order.judgeAndLegalAdvisor.legalAdvisorName);
+  await I.completeEvent('Save and continue');
+};
+
+const createCareOrder = async (I, createOrderEventPage, order) => {
+  await createOrderEventPage.selectType(order.type);
+  await I.retryUntilExists(() => I.click('Continue'), '#judgeAndLegalAdvisor_judgeTitle');
+  await createOrderEventPage.enterJudgeAndLegalAdvisor(order.judgeAndLegalAdvisor.judgeLastName, order.judgeAndLegalAdvisor.legalAdvisorName);
+  await I.retryUntilExists(() => I.click('Continue'), createOrderEventPage.fields.directionsNeeded.id);
+  await createOrderEventPage.enterDirections('example directions');
+  await I.completeEvent('Save and continue');
+};
+
+const createEmergencyProtectionOrder = async (I, createOrderEventPage, order) => {
+  const tomorrow = new Date(Date.now() + (3600 * 1000 * 24));
+
+  await createOrderEventPage.selectType(order.type);
+  await I.retryUntilExists(() => I.click('Continue'), '#epoChildren_descriptionNeeded');
```

Comment:
It would be better to use `createOrderEventPage.fields.epo.childrenDescription.radioGroup`
A bit longer but would mean less changes if stuff in the ccd definitions change